packages/react-cache/src/__tests__/ReactCacheOld-test.internal.js

```diff
@@ -190,20 +192,31 @@ describe('ReactCache', () => {
     );
 
     if (__DEV__) {
-      await expect(async () => {
-        await waitForAll([
-          'App',
-          'Loading...',
-
-          ...(gate('enableSiblingPrerendering') ? ['App'] : []),
-        ]);
-      }).toErrorDev([
+      await waitForAll([
+        'App',
+        'Loading...',
+
+        ...(gate('enableSiblingPrerendering') ? ['App'] : []),
+      ]);
+      assertConsoleErrorDev([
         'Invalid key type. Expected a string, number, symbol, or ' +
           "boolean, but instead received: [ 'Hi', 100 ]\n\n" +
           'To use non-primitive values as keys, you must pass a hash ' +
-          'function as the second argument to createResource().',
-
-        ...(gate('enableSiblingPrerendering') ? ['Invalid key type'] : []),
+          'function as the second argument to createResource().\n' +
+          '    in App (at **)' +
+          (gate(flags => flags.enableOwnerStacks)
+            ? ''
+            : '\n    in Suspense (at **)'),
+
+        ...(gate('enableSiblingPrerendering')
```

Comment:
is this shorthand for `gate(flags => flags.enableSiblingPrerendering)`?

Reply:
yeah, added in aug https://github.com/facebook/react/pull/30760